src/catalog/src/information_schema/region_peers.rs

```diff
@@ -205,18 +205,23 @@ impl InformationSchemaRegionPeersBuilder {
                     table_ids.into_iter().map(|id| (id, vec![])).collect()
                 };
 
-                for routes in table_routes.values() {
-                    self.add_region_peers(&predicates, routes);
+                for (table_id, routes) in table_routes {
```

Comment:
nit:
```suggestion
                for (physical_table_id, routes) in table_routes {
```

Reply:
There maybe `logical table`, `table_id` is more reasonable?

Reply:
It is logical table @WenyXu

Reply:
Good catch, you're right🤩